src/drivers/driver_zephyr.c

```diff
@@ -1546,6 +1596,156 @@ int wpa_drv_zep_deinit_ap(void *priv)
 
 	ret = 0;
 
+out:
+	return ret;
+}
+
+int wpa_drv_zep_sta_add(void *priv, struct hostapd_sta_add_params *params)
+{
+	struct zep_drv_if_ctx *if_ctx = priv;
+	const struct zep_wpa_supp_dev_ops *dev_ops = NULL;
+	int ret = -1;
+
+	if ((!priv) || (!params)) {
+		wpa_printf(MSG_ERROR, "%s: Invalid params\n", __func__);
+		goto out;
+	}
+
+	dev_ops = if_ctx->dev_ctx->config;
+	if (!dev_ops->sta_add) {
+		wpa_printf(MSG_ERROR, "%s: sta_add op not supported\n", __func__);
+		goto out;
+	}
+
+	ret = dev_ops->sta_add(if_ctx->dev_priv, params);
+	if (ret) {
+		wpa_printf(MSG_ERROR, "%s: sta_add op failed: %d\n", __func__, ret);
+		goto out;
+	}
+
+	ret = 0;
```

Comment:
I guess this is redundant ? (Same comment for all subsequent functions added in this commit).